.github/workflows/swiftlint.yml

```diff
@@ -0,0 +1,20 @@
+name: SwiftLint
+
+on:
+  pull_request:
+    paths:
+      - '.github/workflows/swiftlint.yml'
+      - '.swiftlint.yml'
+      - '**/*.swift'
+
+jobs:
+  SwiftLint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: GitHub Action for SwiftLint
+        # Avoid failing with "server error status: 403" on PR from forked repository
```

Comment:
Why do we have this problem just for this action? Seems odd.

Reply:
`norio-nomura/action-swiftlint` uses GitHub Checks APIs with using `GITHUB_TOKEN` for adding annotations from parsed SwiftLint violations.
Since `GITHUB_TOKEN` provided on PR from forked repository does not have write permission to GitHub Checks APIs, it fails with "server error status: 403".
Maybe I should change `norio-nomura/action-swiftlint` to ignore the 403 error.

Reply:
Ah I see. Yeah ideally the SwiftLint action would still run, just would skip posting the checks to a repo it doesn't have permission for.